vignettes/strict.Rmd

```diff
@@ -58,12 +58,11 @@ function()
 1 ~  more()   #   comment
 ```
 
--   More than one line break is tolerated before closing curly brace and line breaks between curly and round braces are not removed.
+-   Line breaks between curly and round braces are not removed.
```

Comment:
I don't think we should be tolerating such stray empty lines in any mode, strict or non-strict. 

Such line breaks are rarely intentional, and are often the result of code being removed via "search and remove".

Reply:
Maybe yes. Can always open an issue for it, but I wont' work on it 😀